packages/clay-sticker/src/ClaySticker.soy

```diff
@@ -72,7 +72,21 @@
 	{@param? imageAlt: string}
 	{@param? imageSrc: string}
```

Comment:
In fact this two params should be mandatory for this `.image` template, so we can get rid of `let` and just place the variables in the image, or at least remove the conditions.